### ISSUE TITLE
TypeScript Port

### ISSUE THREAD

**Issue by pspeter3** [BorisTheBrave/sylves#1]
This library looks amazing and I was wondering if you would be ok with someone porting it to TypeScript?

**Comment by BorisTheBrave** [comment]
Yes, I would love to see that. If it is a port, then you'll need to include the MIT licenses from this project to indicate you've taken material from it it.

If you like, you can use some variant of the Sylves name providing you indicate it is a port of this library. As I intend to continue developing the library, it would also be sensible to indicate what version your port was made from. (currently, [0.1](https://github.com/BorisTheBrave/sylves/releases/tag/v0.1.0))
